test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/meta/main.yml

```diff
@@ -0,0 +1,60 @@
+galaxy_info:
```

Comment:
Please remove this file.

Reply:
will remove that

Reply:
Done. Commit id @322fae10765b04cb0f3f6bbbfb2024f59f7fb924

Reply:
done